src/server/main.py

```diff
@@ -35,10 +37,19 @@
     return templates.TemplateResponse("model.html", {"request": request},)
 
 
-@app.get("/selections")
+@app.get("/request-event-evaluation")
```

Comment:
@hnykda - was this supposed to be replaced?

Reply:
Yes, that's fine.